layouts/BeltDocsLayout.re

```diff
@@ -94,23 +94,48 @@ module Md = {
     );
 };
 
+module FlavourSwitch = {
```

Comment:
This should probably be relocated to `components/FlavourSwitch.re` to prevent coupling of `Text.re` to `BeltDocsLayout.re`